src/components/NcSettingsSelectGroup/NcSettingsSelectGroup.vue

```diff
@@ -111,7 +138,7 @@ export default {
 	},
 	computed: {
 		inputValue() {
-			return this.getValueObject()
+			return this.getValueObject
```

Comment:
This change is only needed to make it usable in general, right?
Would it make sense to backport it to 7.11.x?

Reply:
Yes, I would partially backport this

Reply:
Can you do that? :)

Reply:
@susnux please backport to version 7, then we need to release it and update the version in server stable27 and stable26